src/cryptonote_core/service_node_rules.cpp

```diff
@@ -3,6 +3,7 @@
 #include "int-util.h"
 #include <vector>
 #include <boost/lexical_cast.hpp>
+#include <fenv.h>
```

Comment:
This should be `#include <cfenv>` (the raw C header won't give you the functions in the `std` namespace).

Reply:
Fixed